.buildkite/steps/test-demo-notebooks.sh

```diff
@@ -51,7 +51,8 @@ echo "+++ :python: running $f"
 cd "$(dirname "$f")"
 # run the notebook, saving it back to where it was, printing everything
 exitCode=0
-papermill --execution-timeout=600 --log-output "$f" "$f" || exitCode=$?
+# papermill will replace parameters on some notebooks to make them run faster in CI
+papermill --execution-timeout=600 -p epochs 2 -p walk_length 5 -p batch_size 5 -p n_estimators 2 -p n_predictions 2 --log-output "$f" "$f" || exitCode=$?
```

Comment:
This might be clearer as the YAML format, e.g.:

```bash
parameters='
---
epochs: 2
walk_length: 5
batch_size: 5
n_estimators: 2
n_predictors: 2
'

papermill --execution-timeout=600 --parameters_yaml "$parameters" ...
```

Or, even have it as a separate file like `.buildkite/notebook-parameters.yml` and write:

```bash
papermill --execution-timeout=600 --parameters_file ".buildkite/notebook-parameters.yml" ...
```

Reply:
Using a YAML file keeps it clearest, I think 👍